dist/windows/config.nsh

```diff
@@ -169,10 +136,9 @@ RequestExecutionLevel user
 !define MUI_PAGE_CUSTOMFUNCTION_LEAVE un.check_instance
 !insertmacro MUI_UNPAGE_COMPONENTS
 !insertmacro MUI_UNPAGE_INSTFILES
-
 !insertmacro MUI_RESERVEFILE_LANGDLL
-ReserveFile "${NSISDIR}\Plugins\x86-unicode\FindProcDLL.dll"
-ReserveFile "${NSISDIR}\Plugins\x86-unicode\UAC.dll"
+
+!addplugindir "${QBT_NSIS_PLUGINS_DIR}"
```

Comment:
Maybe this?:
`!addplugindir /x86-unicode "${QBT_NSIS_PLUGINS_DIR}"`

Reply:
Added `/x86-unicode`.Although it should look for Unicode variants by default due to `Unicode true` above.